src/Component-Routes/Feed/Feed.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/jsx-key */
+import './Feed.css'
+import addSign from '../../resource/addSign.png';
+import Pencil from '../../resource/Pencil.png';
+import Trash from '../../resource/Trash.png'
+import { useEffect, useState} from 'react';
+import { collection, onSnapshot} from '../../firebase-store/firebase-imports';
+import { db, auth } from '../../firebase-store/firebaseKeys';
+import { useNavigate } from 'react-router-dom';
+import Header from '../Layout-component/Header';
+import DeleteNote from '../DeleteNote';
```

Comment:
Acomodaría las importaciones por relevancia en el proyecto
```suggestion
import './Feed.css'

import { useEffect, useState} from 'react';
import { useNavigate } from 'react-router-dom';

import { collection, onSnapshot} from '../../firebase-store/firebase-imports';
import { db, auth } from '../../firebase-store/firebaseKeys';
import Header from '../Layout-component/Header';
import DeleteNote from '../DeleteNote';

import addSign from '../../resource/addSign.png';
import Pencil from '../../resource/Pencil.png';
import Trash from '../../resource/Trash.png'
```